server.js

```diff
@@ -451,6 +451,9 @@ module.exports.initExpress = function () {
 
   app.use('/pl/lti', require('./pages/authCallbackLti/authCallbackLti'));
   app.use('/pl/login', require('./pages/authLogin/authLogin'));
+  if (config.devMode) {
+    app.use('/pl/devLogin', require('./pages/authLoginDev/authLoginDev'));
```

Comment:
Micro-nit: consider this for consistency with Azure?

```suggestion
    app.use('/pl/dev_login', require('./pages/authLoginDev/authLoginDev'));
```